staging_notes.md

```diff
@@ -0,0 +1,153 @@
+occurs and then replace the RPC service on port 7777.
+
+### Staging casper-node upgrade
+
+To stage the network protocol upgrade, run:
+
+```bash
+sudo -u casper /etc/casper/node_util.py stage_protocols casper.conf
+```
+
+### Verification of staging
+
+Run the watch command:
+
+```bash
+/etc/casper/node_util.py watch
+```
+
+In the `Next Upgrade` section you should see the 2_0_0 protocol with Era 17889
+
+## Advanced staging
+
+Above is the minimum needed to stage a successful upgrade for 2_0_0 protocol release. This addition section provides notes
+and changes that may be required for a node operator, depending on use cases.
+
+### Modifying config.toml
+
+#### known_addresses
+Some optional changes can be made to `config.toml`.  In this version, the `known_addresses` have been drastically reduced.
+You may wish to take some on your peers and include them as known_addresses. Be careful to include port with ip address (typically 35000).
+Having a varied collection of known_addresses that operators build increases the reliability of the network.
+
+#### allow_request_get_all_values
+
+If you wish to query large auction based data on the node, you must enable a flag for the new `binary_port`. This is disabled
+by default, as queries can be large and consume resources on the node.
+
+```toml
+# Flag that enables the `AllValues` get request. Disabled by default, because it can potentially be abused to retrieve huge amounts of data and clog the node.
+allow_request_get_all_values = true
+```
+
+#### sync_handling
+
+The default for `sync_handling` is `ttl`, which is recommended for validators. If you wish to create a full history
+sync to genesis node, this will continue to take more time as the network grows. Contact Casper association for a link
+to a DB archive which will prepopulate much of this historical data.
+
+There are two main reasons we recommend a validator run as `ttl`:
+
+1. LMDB is a memory backed storage and minimizing the DB footprint will improve performance.
+2. If your node falls behind, with a `genesis` type sync, the node MUST complete historical sync prior to moving 
+ from `KeepUp` to `Validate` mode. Depending on the duration of your outage, this could be some time. Backward sync
+is a background process and de-prioritized compared to running the network.
+
+#### replace.toml
+
+If you have modifications to config.toml that you wish to perform, a `replace.toml` file can be created to assist with this. 
+This file can be used during staging to automate the replace when a `config.toml` file is made from `config-example.toml` during staging.
+
+It should be a minimal version of `config.toml` with only files you wish to replace. And example to change `allow_request_get_all_values`
+is below:
+
+```toml
+[binary_port_server]
+allow_request_get_all_values = true
+```
+
+I prefer to keep `replace.toml` (filename is not important) in `/etc/casper/` with my other config files.
```

Comment:
Let's not write in the first person. Please rephrase.

Reply:
Corrected.